app/views/admin/npcs/index.html.erb

```diff
@@ -1,12 +1,3 @@
 <h1>Welcome <%= current_user.name %>!</h1>
 
-<h2>NPC's:</h2>
-<section class='npcs'>
-  <% if @npcs.nil? || @npcs == [] %>
-    <p>There are currently no NPC's saved.</p>
-  <% else %>
-    <% @npcs.each do |npc| %>
-      <li><%= link_to npc.name, "/npcs/#{npc[:id]}" %></li>
-    <% end %>
-  <% end %>
-</section>
+<%= render "shared/npc_list" %>
```

Comment:
The same lines were being used in two places, so I figured why not just use a shared file!